src/pypgstac/pyproject.toml

```diff
@@ -38,6 +39,9 @@ test = [
     "pystac[validation]==1.*",
     "types-cachetools",
 ]
+build = [
+    "maturin>=1,<2",
+]
```

Comment:
You can define this if you want, but it's superfluous with `build-system` below. If you use any PEP-517 compliant build tool, `maturin` will be installed automatically. E.g.

```bash
pip install -U build
python -m build .
```
will install maturin into the build environment automatically.